sidebars.js

```diff
@@ -293,6 +293,8 @@ module.exports = {
         'api/touch-bar-slider',
         'api/touch-bar-spacer',
         'api/web-request',
+        'api/command-line-switches',
+        'api/command-line',
```

Comment:
These 2 documents get added always in `main` in a new category so moving them to here in the meantime.